### ISSUE TITLE
Create a people-centric view of issues

### ISSUE THREAD

**Issue by thisandagain** [mozilla/build.webmaker.org#9]
For resourcing and even for understanding what an individual is attached to, it would be great to have a people-centric view of planning issues for a specific sprint (milestone).

For example, I currently don't have a simple way to see all issues in a sprint that @cassiemc is attached to (both P1 and P2). We could do this by parsing the RACI section, or even more simply, by extracting all `@mentions` from each issue `body`.

**Comment by cassiemc** [comment]
Yes, if you need a good laugh, here is my current workflow for assigning designers. Can't imagine what @simonwex's process looks like! 

https://www.evernote.com/shard/s313/sh/221a7b97-c2e4-4de2-95a7-fb84f96505ec/885ca98569f06b88a5bb1dad6fe5dd53

Step 1: Figure out and visually group which ticktes are p1 vs p2
Step 2: Remember who was assigned to previous tickets that are being carried through to this sprint
Step 3: Assign p1s
Step 4: Assign p2s
Step 5: Remember which ones I'm 'quality' on
Step 6: Bang my head on the table
=)

**Comment by davidascher** [comment]
we now have the code in a place where we could implement a thing, if we had a design to go from.
